plugin/trino-pinot/src/main/java/io/trino/plugin/pinot/PinotMetadata.java

```diff
@@ -345,6 +345,13 @@ public ConnectorTableProperties getTableProperties(ConnectorSession session, Con
             return Optional.empty();
         }
 
+        // Do not push aggregations down if a grouping column is an array type.
+        // See https://github.com/apache/pinot/issues/8353 for more details.
```

Comment:
might be helpful to expand into something like "Pinot treats each element of array as a grouping key - see <issue>?